examples/node/pdf2png/pdf2png.js

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Hello World pdf converted to png.
+
+var fs = require('fs');
+var assert = require('assert');
+var Canvas = require('canvas');
+
+function nodeCanvasFactory() {}
+nodeCanvasFactory.prototype = {
+	create: function nodeCanvasFactory_create(width, height) {
+		assert(width > 0 && height > 0, 'invalid canvas size');
+	    var canvas = new Canvas(width, height);
```

Comment:
Here, and on many lines below, please make sure that you indent with *two* spaces instead of four.